packages/fast-components/src/progress/progress-ring/progress-ring.styles.ts

```diff
@@ -0,0 +1,72 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../../styles";
+
+export const ProgressRingStyles = css`
+    :host([hidden]) {
+        display: none;
+    }
+
+    ${display("flex")} :host {
+        align-items: center;
+        outline: none;
+        height: calc(var(--height-number) * 1px);
+        width: calc(var(--height-number) * 1px);
+        margin: calc(var(--design-unit) * 1px) 0;
+    }
+
+    .progress {
+        height: 100%;
+        width: 100%;
+    }
+
+    .background {
+        stroke: var(--neutral-fill-rest);
+        fill: none;
+        stroke-width: 2px;
+    }
+
+    .determinate {
+        stroke: var(--accent-fill-rest);
+        fill: none;
+        stroke-width: 2px;
+        stroke-linecap: round;
+        transform-origin: 50% 50%;
+        transform: rotate(-90deg);
+        transition: all 0.2s ease-in-out;
+    }
+
+    .indeterminate-indicator-1 {
```

Comment:
should we use the "-1" on "indeterminate-indicator-1" if there is only one?

Reply:
i did this to keep the structure similar across each progress component and not have duplicated part names.